apps/session-manager/src/services/__tests__/redis-session-storage.spec.ts

```diff
@@ -267,6 +269,110 @@ describe("RedisSessionStorage#getBySessionToken", () => {
   });
 });
 
+describe("RedisSessionStorage#getByFIMSToken", () => {
+  const mockedDependencies = {
+    redisClientSelector: mockRedisClientSelector,
+  };
+
+  test("should get the session with a valid token", async () => {
+    mockGet.mockImplementationOnce((_) => Promise.resolve(mockSessionToken));
```

Comment:
I would suggest this shortcut 😃 
```suggestion
    mockGet.mockResolvedValueOnce(mockSessionToken);
```